relay-server/src/actors/project_buffer.rs

```diff
@@ -398,12 +440,15 @@ impl BufferService {
             .bind(key.sampling_key.to_string())
             .fetch(db).peekable();
 
+            relay_statsd::metric!(counter(RelayCounters::BufferReads) += 1);
```

Comment:
Same suggestion here, should we have a counter and log a single metric instead of doing so on each iteration?

Reply:
Same argument as before: the batch size is 100, so I hope the volume is acceptable.